### PR TITLE
Remove Duplicated Components and Import them from `@opencollective/frontend-components` Instead

### PR DESCRIPTION
**Pull request by SudharakaP**
Related to https://github.com/opencollective/opencollective/issues/6069

I've done some testing with receipts and seems to work fine. Also note that I've used the `next-transpile-modules` since this is an ESM module. Let me know if I've missed anything obvious here. 🤔 

image

### CODE REVIEW

**Comment by SudharakaP** [comment]
@Betree : I don't have access to vercel anymore so cannot see why the deployment has failed here. Could you share with me the logs please and I can take a look. 🤗 🙏🏽

File: .env.local [hunk 1]
```diff
@@ -4,3 +4,4 @@ WEBSITE_URL=http://localhost:3000
 API_URL=http://localhost:3060
 API_KEY=dvl-1510egmf4a23d80342403fb599qd
 LOG_LEVEL=debug
+OPENSSL_CONF=/dev/null
```
Comment (Betree) [inline .env.local:7]: Why?

**Review by Betree — COMMENTED** [review]
The Vercel build failure looks unrelated:
> <span>Error: Node.js Version "12.x" is discontinued and must be upgraded. Please set "engines": { "node": "16.x" } in your `package.json` file to use Node.js 16.</span>

Note that the ESM config stuff is documented in the README https://www.npmjs.com/package/@opencollective/frontend-components.

File: .env.local (hunk 1 above)
Comment (SudharakaP) [reply .env.local]: Sorry, this is a mistake which somehow I changed some months bank and forgot to revert. 🙉

**Comment by SudharakaP** [comment]
> Rebased, everything looks good now apart from the `OPENSSL_CONF` thing above

Thanks for the review. I've corrected the `OPENSSL_CONF` thing. Will be merging ones test pass. 👍🏽

**Review by Betree — APPROVED** [review]
Looking good!

**Pull request merged**